GoogleTestAdapter/Core/Resources.resx

```diff
@@ -290,6 +322,182 @@
+Google Test option: {0}</value>
+    <comment>{0} represents a command-line flag.</comment>
+  </data>
+  <data name="OptionShuffleTestsSeed" xml:space="preserve">
+    <value>Shuffle tests: Seed</value>
+  </data>
+  <data name="OptionShuffleTestsSeedDescription" xml:space="preserve">
+    <value>0: Seed is computed from system time, 1&lt;=n&lt;={0}: The given seed is used. See note of option '{1}'.
+Google Test option: {2}</value>
+    <comment>{0} is an integer representing the maximum value of a seed. {1} represents the localized name of another option. {2} represents a command-line flag.</comment>
+  </data>
+  <data name="OptionTestDiscoveryRegex" xml:space="preserve">
+    <value>Regex for test discovery</value>
+  </data>
+  <data name="OptionTestDiscoveryRegexDescription" xml:space="preserve">
+    <value>If non-empty, this regex will be used to discover the Google Test executables containing your tests.
+Default regex:</value>
+    <comment>Followed by a regular expression</comment>
+  </data>
+  <data name="OptionTestDiscoveryTimeoutInSeconds" xml:space="preserve">
+    <value>Test discovery timeout in s</value>
+    <comment>"s" stands for seconds</comment>
+  </data>
+  <data name="OptionTestDiscoveryTimeoutInSecondsDescription" xml:space="preserve">
+    <value>Number of seconds after which test discovery will be assumed to have failed. 0: Infinite timeout</value>
+  </data>
+  <data name="OptionTestNameSeparator" xml:space="preserve">
+    <value>Test name separator</value>
+  </data>
+  <data name="OptionTestNameSeparatorDescription" xml:space="preserve">
+    <value>Test names produced by Google Test might contain the character '/', which makes VS cut the name after the '/' if the test explorer window is not wide enough. This option's value, if non-empty, will replace the '/' character to avoid that behavior. Note that '\', ' ', '|', and '-' produce the same behavior ('.', '_', ':', and '::' are known to work - there might be more). Note also that traits regexes are evaluated against the tests' display names (and must thus be consistent with this option).</value>
+  </data>
+  <data name="OptionTimestampOutput" xml:space="preserve">
+    <value>Timestamp output</value>
+  </data>
+  <data name="OptionTimestampOutputDescription" xml:space="preserve">
+    <value>If true, a timestamp is added to test and debug output.</value>
+  </data>
+  <data name="OptionTraitsDescription" xml:space="preserve">
+    <value>Allows to override/add traits for testcases matching a regex. Traits are build up in 3 phases: 1st, traits are assigned to tests according to the 'Traits before' option. 2nd, the tests' traits (defined via the macros in GTA_Traits.h) are added to the tests, overriding traits from phase 1 with new values. 3rd, the 'Traits after' option is evaluated, again in an overriding manner.
```

Comment:
"1st", "2nd", "3rd" reads weird to me here, but I will leave it to your consideration if they should be changed or this comment ignored :).